index.html

```diff
@@ -44,5 +45,153 @@ <h4>I'm a Software Developer</h4>
+
+        <main class="main-section">
+            <section class="works-card">
+                <h4>Professional Art <br/> Printing Data</h4>
+                <p>
+                    A daily section of privately personalized reads;
+                    no accounts or sign-ups required. Has been the industry's
+                    standard
+                </p>
+                <ul class="works-ul">
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>ruby</li>
+                </ul>
+                <button type="button">See Project</button>
+            </section>
+            <section class="works-card">
+                <h4>Professional Art <br/> Printing Data</h4>
+                <p>
+                    A daily section of privately personalized reads;
+                    no accounts or sign-ups required. Has been the industry's
+                    standard
+                </p>
+                <ul class="works-ul">
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>ruby</li>
+                </ul>
+                <button type="button">See Project</button>
+            </section>
+            <section class="works-card">
+                <h4>Professional Art <br/> Printing Data</h4>
+                <p>
+                    A daily section of privately personalized reads;
+                    no accounts or sign-ups required. Has been the industry's
+                    standard
+                </p>
+                <ul class="works-ul">
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>ruby</li>
+                </ul>
+                <button type="button">See Project</button>
+            </section>
+            <section class="works-card">
+                <h4>Professional Art <br/> Printing Data</h4>
+                <p>
+                    A daily section of privately personalized reads;
+                    no accounts or sign-ups required. Has been the industry's
+                    standard
+                </p>
+                <ul class="works-ul">
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>ruby</li>
+                </ul>
+                <button type="button">See Project</button>
+            </section>
+            <section class="works-card">
+                <h4>Professional Art <br/> Printing Data</h4>
+                <p>
+                    A daily section of privately personalized reads;
+                    no accounts or sign-ups required. Has been the industry's
+                    standard
+                </p>
+                <ul class="works-ul">
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>ruby</li>
+                </ul>
+                <button type="button">See Project</button>
```

Comment:
- [ ] After fixing the styles I can see 👀 that your **See Project** button has a default color and not following the colors of the Figma design template 👇🏽. Kindly fix this by using the correct color as in the design 👍🏽.
### Figma design:
![image](https://user-images.githubusercontent.com/43172164/205329588-29096b9a-9854-4482-a340-3cab0f300437.png)
### Your design:
![image](https://user-images.githubusercontent.com/43172164/205329634-f88e7111-eedd-4a23-8edd-f7fade294b15.png)